.github/workflows/docker-publish.yml

```diff
@@ -3,10 +3,30 @@ name: Docker
 on:
   push:
     branches: [ "main" ]
-    paths: [Dockerfile, .github/workflows/docker-publish.yml, build_container.sh]
+    paths:
+      - Dockerfile
+      - .github/workflows/docker-publish.yml
+      - build_container.sh
+      - Dockerfile.riscv64
+      - build_kernel_riscv64.sh
+      - build_opensbi_riscv64.sh
+      - build_qemu_system_riscv64.sh
+      - build_rootfs_riscv64.sh
+      - build_finalize_riscv64.sh
+      - start_in_qemu_riscv64.sh
```

Comment:
nit (and happy to hear other's thoughts on this!): How about having all these in a `riscv64` subdirectory? Then we can drop the `_riscv64` suffix from all the files, and here we can just do `riscv64/*` instead of listing all the files.